src/libraries/Microsoft.NETCore.Platforms/pkg/runtime.json

```diff
@@ -560,6 +560,57 @@
         "browser"
       ]
     },
+    "catalyst": {
+      "#import": [
+        "linux"
```

Comment:
(I don't know how RIDs work)

Why is it based on "linux"?  "osx" and "ios" are both "unix".  Shouldn't catalyst be like them?

Reply:
Yes, that's what I get for speed reading Marek's issue ;-)